library/src/main/kotlin/com/connectrpc/protocols/GRPCWebInterceptor.kt

```diff
@@ -188,26 +185,14 @@ internal class GRPCWebInterceptor(
             streamResultFunction = { res ->
                 val streamResult = res.fold(
                     onHeaders = { result ->
-                        val responseHeaders = result.headers.filter { entry -> !entry.key.startsWith("trailer") }
```

Comment:
Only Connect, not the gRPC* protocols have this convention with trailers and the `trailer-` prefix.